user-console/source/org/pentaho/mantle/public/themes/crystal/mantleCrystal.css

```diff
@@ -1283,10 +1283,19 @@ a:active, a:hover {
 }
 
 .users-roles-list {
-  height: 432px;
+  height: 100%;
   width: 200px;
 }
 
+/* targetting IE 10 only */
+.IE10 .pentaho-tab-panel {
+    height:541px!important;
```

Comment:
@pamval I hate using !important as well, but for IE styles in our product, I have had to use !important b/c I was not able to make a more specific style to "win" for IE. But i agree, !important is not good.